nyx_client/nyx_client/client.py

```diff
@@ -669,8 +686,14 @@ def update_data(
             A `Data` instance, containing the updated information.
 
         Raises:
-            requests.HTTPError: If the API request fails.
+            ValueError: When neither download_url or file is provided, or both are provided
```

Comment:
Same as previous - simplified sentence 
```suggestion
            ValueError: When download_url and file are both provided or both missing
```